openfisca_us/parameters/gov/states/va/tax/deductions/income_based_age_deduction.yaml

```
@@ -0,0 +1,29 @@
+age:
+  description: Minimum age a tax payer must be to claim the income-based age deduction
+  values:
+    2021-01-01: 65
+  metadata:
+    unit: year
+    label: VA income-based age deduction threshold
+income:
```

Also please add a `phase_out_rate` parameter, equal to `1` if I'm reading correctly.
```suggestion
phase_out_start:
```